gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -108,6 +117,7 @@
   private HomeScreenViewModel viewModel;
   private MapContainerFragment mapContainerFragment;
   private BottomSheetBehavior<View> bottomSheetBehavior;
+  private PublishSubject<FragmentManager> showFeatureDialogRequests;
```

Comment:
On the fence about whether to call these `requests` or something more concrete like `clicks`. Since I'm undecided I'll leave it to you, but just wanted to raise the doubt/question.

Reply:
Please disregard for now, since renaming this would require renaming the method and usages, which is probably out of scope of this PR.

Reply:
Okay, pocketing for later.